scripts/installation/install_solc.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+import platform
+import shutil
+from os.path import dirname, join, abspath
+from os import stat, chmod
+import requests
+
+
+if platform.system() != 'Linux':
+    raise EnvironmentError("This installation script is only compatible with linux-gnu-based operating systems.")
+
+PACKAGE_NAME = join('nucypher', 'blockchain', 'eth', 'sol')
+BASE_DIR = dirname(dirname(dirname(abspath(__file__))))
+FILE_PATH = join(BASE_DIR, PACKAGE_NAME, "__init__.py")
```

Comment:
I am having second thoughts about using `__init__.py` as the location since it implies that *no new code* can be added there without also considering the `exec`.  What do you think about using the same path, but the name `__version__.py` instead?

Reply:
`__version__.py` looks like version of sub-module. Maybe `conf.py`?

Reply:
I think the dunder is warranted - `__conf__.py?`

Reply:
👍 
✔️